toolchain/check/operator.h

```diff
@@ -0,0 +1,31 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_CHECK_OPERATOR_H_
+#define CARBON_TOOLCHAIN_CHECK_OPERATOR_H_
+
+#include "toolchain/check/context.h"
+#include "toolchain/parse/node_ids.h"
+#include "toolchain/sem_ir/ids.h"
+
+namespace Carbon::Check {
+
+struct Operator {
+  llvm::StringLiteral interface_name;
+  llvm::StringLiteral op_name = "Op";
+};
+
+// Checks and builds SemIR for a unary operator expression, `$operand` or
+// `operand$`.
```

Comment:
```suggestion
// Checks and builds SemIR for a unary operator expression. For example, `$operand` or
// `operand$`.
```

I think clearly indicating this is an example would help avoid confusion. You might also consider replacing `$` with an operator Carbon uses, but maybe my mind is too much on the shell script.